### PR TITLE
Added HackTX Open Source Dashboard

### PR DESCRIPTION
**Pull request by tbarn**
The HackTX team is a great example of an organization who cares about more than just one event but building a collaborative community that helps other students

They are looking to answer questions about what is going on within their multiple open source projects

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -13,6 +13,8 @@
 * From the [CMX](http://cmxsummit.com/) Yearbook project (a community/sponsorship effort by the Keen IO team):
   * [Landing page and dashboard](http://dashboards.community/)
 
+* From [HackTX](http://hacktx.com/) and [Taylor Barnett](https://twitter.com/taylor_atx):
+  * [HackTX's Open Source Dashboard](bit.ly/hacktxkeen) - created to help display organization level GitHub metrics
```
Comment (leggetter) [inline examples.md]: Missing `https://` on the `bit.ly/hacktxkeen` link

Comment (tbarn) [inline examples.md]: Oops! Bad copy/paste. Fixed.

**Comment by heitortsergent** [comment]
:+1: :shipit:

**Comment by tbarn** [comment]
Thanks @heitortsergent! I was about to bug someone for a +1 soon.

**Comment by timfalls** [comment]
woot!

**Pull request merged**